rflx/expression.py

```diff
@@ -710,20 +711,14 @@ def z3expr(self) -> z3.ArithRef:
 
 
 class Name(Expr):
```

Comment:
The class `Name` doesnt't seem to have a name anymore. Should it be renamed into something else then?

Reply:
`Name` is now an abstract base class. The unique feature of all from `Name` derived classes is that they can be substituted (by using `simplified`). I have no good idea for a better name, but I am open to suggestions

Reply:
Would `Token` be a viable name?

Reply:
I wouldn't associate this functionality with `Token`. When I hear tokens, I rather think of very basic entities like symbols.

The origin of `Name` is the Ada RM: http://www.ada-auth.org/standards/2xrm/html/RM-4-1.html. I think we should just stick with it.

Reply:
Okay, in this case we should keep the name.